concepticondata/conceptlists/Lai-2023-291.tsv

```diff
@@ -0,0 +1,292 @@
+ID	NUMBER	ENGLISH	CONCEPTICON_ID	CONCEPTICON_GLOSS	HUANG_1992_1820	LEXIBANK_GLOSS
+Lai-2023-291-1	41	the dust	2	DUST	45	the dust
+Lai-2023-291-2	55	firewood	10	FIREWOOD	534	firewood
+Lai-2023-291-3	145	noon	12	MIDDAY	750	noon // the noon
+Lai-2023-291-4	149	the otter	15	OTTER	317	the otter
+Lai-2023-291-5	208	that	78	THAT	947	that
+Lai-2023-291-6	2	all	98	ALL	962	all
+Lai-2023-291-7	265	here	136	HERE		here
+Lai-2023-291-8	372	to burn [transitive]	141	BURN (SOMETHING)		to burn [transitive]
+Lai-2023-291-9	92	the hoof	152	HOOF	264	the hoof
+Lai-2023-291-10	158	red	156	RED	1007	red
+Lai-2023-291-11	14	black	163	BLACK	1005	black
+Lai-2023-291-12	16	to blow (of wind)	175	BLOW (OF WIND)	1738	to blow (of wind)
+Lai-2023-291-13	17	the body hair (hair or fur)	189	HAIR (BODY)	266	the body hair (hair or fur) // the hair
```

Comment:
Perhaps, [HAIR OR FUR](https://concepticon.clld.org/parameters/2678) is better for this one.

Reply:
The Chinese concept is maximally unspecified, but in this case, the base list links already to body hair, so we better just follow that mapping.